pkg/test/resourcefixture/testdata/basic/apikeys/v1alpha1/apikeyskey/apikeyskeybasic/create.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apikeys.cnrm.cloud.google.com/v1alpha1
+kind: APIKeysKey
+metadata:
+  name: apikeyskeybasic-${uniqueId}
```

Comment:
Should be `name: aapikeyskey-${uniqueId}`. The naming convention is `[kindinlowercase]-${uniqueId}`.

Reply:
Not sure I agree with this convention (we know the kind from the kind, so we get more information from [test name]-$uniqueId), but ... fixed